src/index.js

```diff
@@ -50,6 +52,13 @@ function volunteerWithCustomFields(volunteerAndDistance) {
 }
 
 // Accepts errand address and checks volunteer spreadsheet for closest volunteers
+/**
+ * Find volunteers.
+ *
+ * @param {object} request The Airtable request object.
```

Comment:
The default JSDoc config does not allow for unknown types, so fails with this error:

     The type 'RecquestRecord' is undefined. [Warning/jsdoc/no-undefined-types]

We might be able to define the type with [@typedef](https://jsdoc.app/tags-typedef.html) but I'd have to investigate how that would work here, since the source is coming from a vendor.

Would `object` be okay for now?

Reply:
Ah I see. Yep, object is good for now!